docs/provider-configuration.md

```diff
@@ -261,6 +261,9 @@ and should appear in the `[BlockStorage]` section of the `$CLOUD_CONFIG` file.
 
 * `node-volume-attach-limit`: Maximum volumes that can be attached to the node. Its
   default value is 256.
+* `rescan-on-resize`: Rescan block device and verify its size before expanding the
```

Comment:
@kayrus please mention here , it is required (must enabled) only for VMWare as of now , unless its tested for other hyper-visors, can't be generalized.

Reply:
Done